knimin/lib/format.py

```diff
@@ -71,6 +71,33 @@ def format_normalization_echo_pick_list(vol_sample, vol_water):
     return "\n".join(contents)
 
 
+def format_pooling_echo_pick_list(vol_sample):
+    """Format the contents of an echo pooling pick list file_name
+
+    Parameters
+    ----------
+    vol_sample : 2d numpy array of floats
+        The per well sample volume
+    """
+    contents = ['Source Plate Name,Source Plate Type,Source Well,'
+                'Concentration,Transfer Volume,Destination Plate Name,'
+                'Destination Well']
+    # Write the sample transfer volumes
+    rows, cols = vol_sample.shape
+    for i in range(rows):
+        for j in range(cols):
+            well_name = "%s%d" % (chr(ord('A') + i), j+1)
+            # Machine will round, so just give it enough info to do the
+            # correct rounding. The desination well is A1 because we're pooling
+            # samples
+            val = "%.2f" % vol_sample[i][j]
```

Comment:
This should be in nL, and the machine has a resolution of 2.5 nL. So two digits after the dot is great.

Probably a good idea to include units in comments as frequently as possible to avoid issues.